index.js

```diff
@@ -17,7 +17,7 @@ var isURL = require('./lib/isURL.js');
 var sheetsuNode = function(config) {
   var configParam = config || {};
 
-  configParam.version = configParam.version || '1.0';
+  configParam.version = configParam.version || '1.0on';
```

Comment:
what does the "o" letter mean?

Reply:
Update docs plz
https://docs.google.com/a/sheetsu.com/spreadsheets/d/1kTRZcOTF0V7ye4I4WzPgf2E2ajQ1HaLcVG5-9rsNQyc/edit?usp=sharing